template-parts/person/at_a_glance.php

```diff
@@ -46,12 +45,25 @@
 								</dd>
 								<?php endif; ?>
 
-								<?php if ( $department ) : ?>
-								<dt class="h6 text-uppercase text-default mb-3">Department</dt>
+								<?php if ( $departments ) : ?>
+								<dt class="h6 text-uppercase text-muted mb-2">Department(s)</dt>
 								<dd class="h5 mt-2 mb-4">
-									<span class="d-block">
-										<?php echo $department; ?>
-									</span>
+									<?php
+									foreach ( $departments as $department ) :
+										$department_url = get_term_link( $department->term_id );
```

Comment:
If we do link to individual departments here, we will likely want to link out elsewhere--maybe to faculty search results?

If we do go that route, you can grab the faculty search URL with `get_faculty_search_page_url()` and just stick the department slug at the end as a query param (`"?department={$department->slug}"`)

Reply:
We decided to remove links to departments, updated here: 
https://github.com/UCF/Main-Site-Theme/pull/381/commits/60b77dba4b170f4fa1c417e81a58d552ef3a7562